tests/common/plugins/sanity_check/checks.py

```diff
@@ -278,6 +285,166 @@ def _check_monit_services_status(check_result, monit_services_status):
+
+        lower_tor_host.shell(ping_cmd.format(tor_mux_intf, lower_tor_ping_tgt_ip))
+        try:
+            testutils.verify_no_packet(ptfadapter, lower_tor_exp_pkt, ptf_port_index)
+        except AssertionError:
+            results['failed'] = True
+            results['failed_reason'] = 'Packet from standby lower ToR received'
+            return results
+
+        # Send dummy ARP packets from PTF to ToR. Ensure that ARP is learned on active ToR only
+        upper_tor_host.shell("ip neigh flush all")
+        lower_tor_host.shell("ip neigh flush all")
+        testutils.send_packet(ptfadapter, ptf_port_index, ptf_arp_pkt)
+
+        upper_tor_arp_table = upper_tor_host.switch_arptable()['ansible_facts']['arptable']['v4']
+        lower_tor_arp_table = lower_tor_host.switch_arptable()['ansible_facts']['arptable']['v4']
+        try:
+            pytest_assert(ptf_arp_tgt_ip in upper_tor_arp_table)
+        except AssertionError:
+            results['failed'] = True
+            results['failed_reason'] = 'Packet from PTF not received on active upper ToR'
+            return results
+
+        try:
+            pytest_assert(ptf_arp_tgt_ip in lower_tor_arp_table)
+        except AssertionError:
+            results['failed'] = True
+            results['failed_reason'] = 'Packet from PTF not received on standby lower ToR'
+            return results
+
+        # Repeat all tests with lower ToR active
+        toggle_simulator_port_to_lower_tor(mux_server_url, mux_simulator_index)
+        try:
+            pytest_assert(get_simulator_read_side(mux_server_url, mux_simulator_index) == 2)
+        except AssertionError:
+            results['failed'] = True
+            results['failed_reason'] = 'Unable to switch active link to lower ToR'
+            return results
+
+        lower_tor_host.shell(ping_cmd.format(tor_mux_intf, lower_tor_ping_tgt_ip))
+        try:
+            testutils.verify_packet(ptfadapter, lower_tor_exp_pkt, ptf_port_index)
+        except AssertionError:
+            results['failed'] = True
+            results['failed_reason'] = 'Packet from active lower ToR not received'
+            return results
+
+        upper_tor_host.shell(ping_cmd.format(tor_mux_intf, upper_tor_ping_tgt_ip))
+        try:
+            testutils.verify_no_packet(ptfadapter, upper_tor_exp_pkt, ptf_port_index)
+        except AssertionError:
+            results['failed'] = True
+            results['failed_reason'] = 'Packet from standby upper ToR received'
+            return results
+
+        upper_tor_host.shell("ip neigh flush all")
+        lower_tor_host.shell("ip neigh flush all")
+        testutils.send_packet(ptfadapter, ptf_port_index, ptf_arp_pkt)
+
+        upper_tor_arp_table = upper_tor_host.switch_arptable()['ansible_facts']['arptable']['v4']
+        lower_tor_arp_table = lower_tor_host.switch_arptable()['ansible_facts']['arptable']['v4']
+        try:
+            pytest_assert(ptf_arp_tgt_ip in upper_tor_arp_table)
```

Comment:
`ptf_arp_tgt_ip in upper_tor_arp_table` --> `ptf_arp_tgt_ip not in upper_tor_arp_table`?

Reply:
See comment https://github.com/Azure/sonic-mgmt/pull/2862#discussion_r579359928